descriptors/ModuleDescriptor-template.json

```diff
@@ -8,7 +8,7 @@
       "handlers": [
         {
           "methods": [ "POST" ],
-          "pathPattern": "/resource",
+          "pathPattern": "/linked-data/resource",
```

Comment:
Let us rename the "id" in line 6 to "linked-data".
Also, remove the providers with id "profile", "reindex" and "graph" and move everything under "linked-data" provider.

Reply:
done